rest/sgcollect_test.go

```diff
@@ -117,12 +118,15 @@ func TestSgcollectOptionsValidateInvalid(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.name, func(ts *testing.T) {
 			errs := test.options.Validate()
+			require.NotNil(t, errs)
+			multiError, ok := errs.(*multierror.Error)
+			require.True(t, ok)
 
 			// make sure we get at least one error for the given invalid options.
-			assert.True(t, len(errs) > 0)
+			require.NotNil(t, multiError.ErrorOrNil())
```

Comment:
Since we've already done a nil check on line 121, can we check len(multiError.Errors) here?

Reply:
Sure, length check can be done here and that is ideal.